examples/mockups/index.html

```diff
@@ -0,0 +1,329 @@
+                <li class="head">Contributing</li>
+                <li><a href="https://docs.readthedocs.io/en/stable/contribute.html">to Read the Docs</a></li>
+                <li><a href="https://sphinx-rtd-theme.readthedocs.io/en/latest/contributing.html">to Our Sphinx theme</a></li>
+              </ul>
+              
+              <ul>
+                <li class="head">Support</li>
+                <li><a href="#">Github support guidelines</a></li>
+                <li><a href="https://readthedocs.org/support/">Site support</a></li>
+              </ul>
+
+            </div>
+
+            
+            <div class="marketing">
+
+              <div class="newsletter">
+                <h4 class="head">Stay updated</h4>
+                <p>Sign up for our newsletter to get our latest <a href="https://blog.readthedocs.com/">blog</a> updates delivered to your inbox weekly.</p>
+
+                <div class="ui input action fluid inverted">
+                  <input type="text" placeholder="email@host.com">
+                  <div class="ui button primary">Subscribe</div>
+                </div>
+              </div>
+
+              <div class="social">
+                <i class="icon github"></i>
+                <i class="icon twitter"></i>
+              </div>
+
+            </div>
+
+            <div class="logo">
+              <img src="./images/logo-light.svg"/>
+            </div>
+
+          </div>
+          
+          <div class="sponsors">
+
+            <a class="sponsor" href="#"><img src="./images/sponsors/sponsor-logo-light.svg" alt="Sponsor Name"></a>
+            <a class="sponsor" href="#"><img src="./images/sponsors/sponsor-logo-light.svg" alt="Sponsor Name"></a>
+            <a class="sponsor" href="#"><img src="./images/sponsors/sponsor-logo-light.svg" alt="Sponsor Name"></a>
+            <a class="sponsor" href="#"><img src="./images/sponsors/sponsor-logo-light.svg" alt="Sponsor Name"></a>
+            <a class="sponsor" href="#"><img src="./images/sponsors/sponsor-logo-light.svg" alt="Sponsor Name"></a>
+            <a class="sponsor" href="#"><img src="./images/sponsors/sponsor-logo-light.svg" alt="Sponsor Name"></a>
+
+            <a class="sponsor" href="#"><img src="./images/sponsors/sponsor-logo-light.svg" alt="Sponsor Name"></a>
+            <a class="sponsor" href="#"><img src="./images/sponsors/sponsor-logo-light.svg" alt="Sponsor Name"></a>
+            <a class="sponsor" href="#"><img src="./images/sponsors/sponsor-logo-light.svg" alt="Sponsor Name"></a>
+            <a class="sponsor" href="#"><img src="./images/sponsors/sponsor-logo-light.svg" alt="Sponsor Name"></a>
+            <a class="sponsor" href="#"><img src="./images/sponsors/sponsor-logo-light.svg" alt="Sponsor Name"></a>
+            <a class="sponsor" href="#"><img src="./images/sponsors/sponsor-logo-light.svg" alt="Sponsor Name"></a>
+
+          </div>
+
+        </div>
+      </section>
+
+      
+      <section class="copyrights">
```

Comment:
I'll remake this together with the rest of the footer.
I'd still prefer to keep the naming, as I use it for semantic separation, if nothing else, and it might come handy for JavaScript purposes. As all other similar occurrences, it might make more sense to turn these into ID's.

Reply:
Also done with the app forked footer.